sdk/src/types/block/output/mod.rs

```diff
@@ -416,3 +435,15 @@ pub trait MinimumOutputAmount: StorageScore {
         self.storage_score(params) * params.storage_cost()
     }
 }
+
+/// Decayed stored and potential Mana of an output.
+#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Getters)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct DecayedMana {
```

Comment:
Remind me to use this in the balance PR once this is merged

Reply:
Reminder